bundle/LegacyMapper/Configuration.php

```diff
@@ -277,10 +277,21 @@ private function getImageSettings()
     private function getMultiSiteSettings()
     {
         $rootLocationId = $this->configResolver->getParameter( 'content.tree_root.location_id' );
+        $indexPage = $this->configResolver->getParameter( 'index_page' );
         $defaultPage = $this->configResolver->getParameter( 'default_page' );
         if ( $rootLocationId === null )
         {
-            return array();
+            // return SiteSettings if there is no MultiSite (rootLocation is not defined)
+            $result = array();
+            if ( $indexPage )
```

Comment:
The reasoning behind this as is, is that the setting should already be valid if set, otherwise there it would be an issue in the configuration parser, right?
In any case it could be changed to something like is_string, not empty?